scripts/build.py

```diff
@@ -580,20 +683,22 @@ def build_files(source, target, file_hashes, file_formats=None):
         target: str. Path relative to /oppia directory of directory where
             to copy the files and directories.
         file_hashes: dict(str, str). Dictionary of file hashes.
+        build_tasks: deque(thread). A deque that contains all build tasks queued
```

Comment:
Here and elsewhere, just to double-check: is the type `thread` or `Thread`?

Reply:
It is of type `thread.Thread()`, should it be `Thread`?

Reply:
Yes, Thread is correct.

Reply:
Done!